FSI_Validation/setup.py

```diff
@@ -51,10 +51,10 @@
                     Extension("Chrono_Proteus_Flex",['Chrono_Proteus_Flex.pyx'],
                              depends=['Chrono_Proteus_Flex.h','outputUtil.h'],
                              language='c++',
-                             include_dirs=[numpy.get_include(),'proteus',config.PROTEUS_INCLUDE_DIR],
+                             include_dirs=[numpy.get_include(),'proteus',config.PROTEUS_INCLUDE_DIR,os.path.join(config.PROTEUS_INCLUDE_DIR,'eigen3')],
                              library_dirs=[config.PROTEUS_LIB_DIR,config.PROTEUS_LIB_DIR[:-3]+'lib64'],
                              libraries=['ChronoEngine',
                                         'stdc++','m'],
-                             extra_compile_args=["-std=c++11"]),
+                             extra_compile_args=["-std=c++17"]),
```

Comment:
@Milad-Rakhsha, this appears to have been the source of the problem I was having. More here:
http://eigen.tuxfamily.org/dox-devel/group__TopicUnalignedArrayAssert.html

I'm running the FSI_Validation problem now.

/cc @DanNegrut

Reply:
That's great thanks for figuring this out! I think back when I was working on this Chrono was not employing Eigen so using c++11 standards was just fine, but I am glad that everything looks to be working fine again!